<!DOCTYPE html>
<html>

<head>
  <title>Sophie Furlow - Homepage</title>
  <link rel="stylesheet" href="./assets/css/styles.css">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
  <link rel="preconnect" href="https://fonts.googleapis.com">
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
  <link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@300..700&display=swap" rel="stylesheet">
</head>

<body>
  <header>
    <div class="menu">
      <ul>
        <li><a href="#" onclick="showSection('aboutme')">About Me</a></li>
        <li><a href="#" onclick="showSection('education')">Education</a></li>
        <li><a href="#" onclick="showSection('projects')">Projects</a></li>
        <li><a href="#" onclick="showSection('skills')">Skills</a></li>
      </ul>
    </div>
  </header>
  <div class="sidebar">
    <img src="images/calid.jpg" alt="Sophie Furlow">
    <h1>Sophie Furlow</h1>
    <p>Abbott Statistician</p>
    <p>MEng, Berkeley Bioengineering</p>
    <a href="mailto:sophiefurlow@gmail.com" class="contact-button">CONTACT ME</a>
    <ul class="socials">
      <li><a href="https://github.com/sfurlow8" target="_blank">
          <i class="fab fa-github"></i>
        </a></li>
      <li><a href="http://www.linkedin.com/in/sophie-furlow" target="_blank">
          <i class="fab fa-linkedin"></i>
        </a></li>
    </ul>
  </div>

  <div class="main-content">
    <section id="aboutme" class="section">
      <h2>About Me</h2>
      <p>I am currently a statistician at Abbott in the Diagnostics division. I recently graduated with an MEng in Bioengineering 
        from Berkeley, concentrating on Bioinformatics and Computational Biology. Some of my interests are in genomics, 
        natural language processing (NLP), data engineering, and bioinformatics.
        Beyond tech, I love to draw, explore cities, and spend time with my dogs. </p>
      <p>Please feel free to reach out to me at sophiefurlow [at] gmail [dot] com. Thanks!</p>
    </section>
    <section id="education" class="section">
      <h3 class="school">University of California, Berkeley</h3>
      <h2 class="degree">Master of Engineering, Bioengineering</h2>
      <h4>Berkeley, CA. 2023-2024.</h4>
      <p>Concentration on Bioinformatics and Computational Biology.</p>
      <p>Relevant coursework:</p>
      <ul class="conc">
        <li>computational and functional genomics</li>
        <li>machine learning and optimization for computational biology</li>
        <li>natural language processing</li>
        <li>genome engineering</li>
        <li>business of healthcare</li>
        <li>communication for engineering leaders</li>
      </ul>
      <h3 class="school">Northwestern University</h3>
      <h2 class="degree">Bachelor of Science, Biomedical Computation</h2>
      <h4>Evanston, IL. 2019-2023.</h4>
      <p>Minor in Data Engineering. Magna Cum Laude Graduate. Tau Beta Pi Engineering Honor Society.</p>
      <p>Relevant coursework:</p>
      <ul>
        <li>optimization</li>
        <li>statistics and machine learning</li>
        <li>data structures and algorithms</li>
        <li>data engineering & development</li>
        <li>linear algebra & differential equations</li>
        <li>molecular & cell biology</li>
        <li>genetics</li>
      </ul>
    </section>
    <section id="projects" class="section">
      <h2>Projects</h2>
      <p></p>
      <div class="project" id="coma">
        <div class="project-content">
          <h3>Predicting Coma Recovery with Multimodal Machine Learning</h3>
          <p>ML, NLP, Python</p>
        </div>
      </div>
      <div class="project" id="yeast">
        <div>
          <h3>Transcriptional Analysis of Stress Responses in Yeast</h3>
          <p>Unsupervised learning, transcriptomics</p>
        </div>
      </div>
      <div class="project" id="pd">
        <div class="project-content">
          <h3>At-Home Screening for Parkinson's Disease</h3>
          <p>App development, Swift</p>
        </div>
      </div>
      <div class="project" id="dataeng">
        <div class="'project-content">
          <h3>Data Engineering Pipeline Design</h3>
          <p>AWS, Airflow, Pyspark, PostgreSQL</p>
        </div>
      </div>
      <div class="project" id="tumor">
        <div class="'project-content">
          <h3>DenseNet Classification of Tumor Images</h3>
          <p>DL, image segmentation</p>
        </div>
      </div>

      <div class="modal-container" id="modal-container-coma">
        <div class="modal">
          <button id="close-coma" style="float: right;">x</button>
          <h1>Predicting Coma Recovery with Machine Learning of Multimodal Data</h1>
          <h2>UC Berkeley and UC San Francisco, 2023-2024</h2>
          <div class="container">
            <p>The standard of care in ICUs is to observe comatose patients for several days before making treatment
              decisions, but early action can lead to better recovery outcomes. We are deploying a model trained on
              historical patients’ neurophysiological data and doctor-written notes to predict the likely recovery
              outcome of new patients. The prediction is a decision support tool that helps physicians make earlier
              treatment decisions to help patients reach a higher level of functional independence post-coma. Using
              Python, we cluster the similarity of brainwave patterns and use natural language processing to correlate
              keywords in patients’ notes to their likely recovery outcomes.
              This work is in the process of publication.</p>
            <img src="images/visual_asset.jpg" alt="coma_visual">
          </div>

          <p><a href="https://github.com/sfurlow8/NLP-of-Medical-Notes" target="_blank">Github</a></p>
          <p><a href="images/brief.pdf" target="_blank">Project brief</a></p>
        </div>
      </div>

      <div class="modal-container" id="modal-container-yeast">
        <div class="modal">
          <button id="close-yeast" style="float: right;">x</button>
          <h1>Transcriptional Analysis of Stress Responses in Yeast</h1>
          <h2>Northwestern University and NSF Simons Center for Quantitative Biology, 2021</h2>
          <p>The goal of the project was to investigate multi-kinase control of environmental stress responsive
            transcription in yeast. What kinases transmit stress signals and what roles of theirs are contingent on
            the environment? My research sought to build on research presented in Mace PLOS 2020 that showed how
            kinases are involved in regulating genes in particular environments.
            I aligned RNA-seq transcripts to a yeast reference genome using STAR in a computing cluster and analyzed them 
            using DESeq2. Using GO analysis and further research on the genotypes clustered together in PCA-reduced space,
            I created an ontological tree of molecular functions. I reasoned that the structure revealed in the PCA plot 
            of the first 2 components forms due to common nucleic binding activity.
          </p>
          <img src="images/yeast_img.jpg" alt="pdscreen" id="yeast_img">
          <p><a href="https://github.com/sfurlow8/Transcriptional-Analysis-of-Stress-Responses"
              target="_blank">Github</a></p>
        </div>
      </div>

      <div class="modal-container" id="modal-container-pd">
        <div class="modal">
          <button id="close-pd" style="float: right;">x</button>
          <h1>At-Home Screening for Parkinson's Disease</h1>
          <h2>UC Berkeley, 2024</h2>
          <div class="container">
            <p>Here we built PDScreen, a mobile app designed to screen at-risk patients for symptoms of Parkinson's at
              home. This app prototype combines a questionnaire, speech recognition, finger tapping, and gait analysis.
              The app is intended to perform parts of the UPDRS (Unified Parkinson’s Disease Rating Scale) screening and
              relay the information to trained neurologists.
              The project was developed over a 2-week period as part of the iHackHealth Appathon hosted by UC Berkeley
              and Apple.
            </p>
            <img src="images/pdscreen.jpg" alt="pdscreen" id="pd_img">
          </div>
          <p><a href="https://github.com/floriankri/PDScreen_iHackHealth_2024" target="_blank">Github (with app
              demo)</a></p>
          <p><a href="images/pd_pitch.pdf" target="_blank">Pitch deck</a></p>
        </div>
      </div>
  </div>

  <div class="modal-container" id="modal-container-dataeng">
    <div class="modal">
      <button id="close-dataeng" style="float: right;">x</button>
      <h1>Data Engineering Pipeline Design</h1>
      <h2>Northwestern University, 2023</h2>
      <p>I completed this project with one team member in order to become comfortable with the ETL workflow using
        PostgreSQL in AWS.
        We set up a pipeline in Airflow that took webscraped information and a dataset of patients' cardiovascular
        symptoms and performed processing and feature engineering using PySpark and MapReduce.
        The processed data was fed into a Logistic Regression, Random Forest, and SVM models with cross validation for
        evaluation on a test set to pick the best model. </p>
      <img src="images/dag.jpg" alt="airflow dag" id="dag_img">
      <p><a href="https://github.com/sfurlow8/Data-Eng-Pipeline-Design/tree/main" target="_blank">Github</a></p>
    </div>
  </div>

  <div class="modal-container" id="modal-container-tumor">
    <div class="modal">
      <button id="close-tumor" style="float: right;">x</button>
      <h1>DenseNet Classification of Tumor Images</h1>
      <h2>UC Berkeley, 2024</h2>
      <p>One team member and I are using images and masks from the NuCLS crowdsourced dataset to train a neural network
        that classifies slide images of cells into tumors, sTIL cells, and stromal cells.
        We have written a DenseNet architecture with 6 dense blocks that can learn multiclass and binary classification
        tasks with up to 85% accuracy on validation sets.
        We run training and validation for the model with roughly 9 million parameters on a high performance computing
        cluster. Images like the one shown below are cropped
        into smaller images containing single nuclei on which the model is trained.</p>
      <img src="images/dense.jpg" alt="tumor cells in slide" id="dense_img">
    </div>
  </div>

  </section>

  <section id="skills" class="section">
    <h2>Skills</h2>
    <section class="skill_cat">
      <h3>Languages</h3>
      <ul>
        <li>Python</li>
        <li>R</li>
        <li>MATLAB</li>
        <li>SQL</li>
        <li>Bash scripting</li>
        <li>JavaScript</li>
        <li>HTML</li>
        <li>Basic C, C++</li>
      </ul>
    </section>
    <section class="skill_cat">
      <h3>Data science</h3>
      <ul>
        <li>Natural Language Processing</li>
        <li>Machine learning</li>
        <li>Deep learning</li>
        <li>PyTorch</li>
        <li>TensorFlow</li>
        <li>PySpark, MapReduce</li>
        <li>Bioconductor</li>
        <li>Biostatistics</li>
        <li>Genome alignment and genomics</li>
      </ul>
    </section>
    <section class="skill_cat">
      <h3>Data management</h3>
      <ul>
        <li>AWS, Azure, GCP</li>
        <li>BigQuery</li>
        <li>Docker</li>
        <li>PostgreSQL</li>
        <li>Airflow</li>
        <li>Git</li>
        <li>High performance computing</li>
      </ul>
    </section>
    <section class="skill_cat">
      <h3>Communication & Creative</h3>
      <ul>
        <li>Presentations & public speaking</li>
        <li>Design methodologies</li>
        <li>Prototyping</li>
        <li>Writing</li>
        <li>Adobe Illustrator</li>
      </ul>
    </section>
</section>
  </div>

  <script src="script.js"></script>
</body>

</html>